py_rql/operators.py

```diff
@@ -54,6 +54,10 @@ def ilike(a, b):
     return like(a.lower(), b.lower())
 
 
+def range_op(a, b):
+    return a >= b[0] and a <= b[1]
```

Comment:
I suggest here a more idiomatic way to do this comparison as `b[0] <= a <= b[1]`

Reply:
I updated the comparison